api/src/certification/complementary-certification/domain/usecases/attach-badges.js

```diff
@@ -51,30 +51,23 @@ const attachBadges = async function ({
       domainTransaction,
     });
 
-    for (const badgeToAttach of complementaryCertificationBadgesToAttach) {
-      const complementaryCertificationBadge = BadgeToAttach.from({
-        ...badgeToAttach,
-        complementaryCertificationId,
-        userId,
-      });
-      await _attachNewComplementaryCertificationBadge({
-        complementaryCertificationBadgesRepository,
-        complementaryCertificationBadge,
-        domainTransaction,
-      });
-    }
+    await _attachNewComplementaryCertificationBadges({
+      complementaryCertificationBadgesRepository,
+      complementaryCertificationBadges: complementaryCertificationBadgesToAttach,
```

Comment:
Pinaillage & inclus de renommer la variable ligne 31
```suggestion
      complementaryCertificationBadges,
```